css/style.css

```diff
@@ -0,0 +1,308 @@
+  color: #fff;
+  font-family: 'cocogoose', sans-serif;
+  font-size: 20px;
+  text-decoration: none;
+  margin: 3px 50px;
+}
+
+#nav-btn {
+  border: 4px solid #ec5242;
+  margin-left: 5px;
+}
+
+.color {
+  color: #272a31;
+}
+
+.color1 {
+  color: #ec5242;
+}
+
+.color2 {
+  color: #d3d3d3;
+}
+
+.no-scroll {
+  overflow: hidden;
+}
+
+.active {
+  color: #ec5242;
+}
+
+.first-container {
+  background-image: url('../images/bgimgfinal.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  text-align: center;
+  padding: 30px;
+}
+
+.font-bold {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+}
+
+footer img {
+  width: 100px;
+  margin: 20px 40px;
+}
+
+.first-container .information {
+  justify-content: center;
+}
+
+.first-container .title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+}
+
+.first-container .heading1 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+}
+
+.first-container .event-details {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  color: #272a31;
+}
+
+.first-container .event-details .first-para {
+  border: 5px solid white;
+  font-size: 18px;
+}
+
+.first-container .info-home {
+  color: #ec5242;
+}
+
+.first-container .info-home .heading1 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 60px;
+}
+
+.first-container .event-details h3 {
+  font-family: 'Lato', sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+}
+
+.first-container .event-details .venue {
+  font-size: 20px;
+}
+
+.up-footer {
+  height: 250px;
+}
+
+.down-footer {
+  background: #d3d3d3;
+}
+
+.down-footer img {
+  width: 200px;
+  margin: 0;
+}
+
+.down-footer h4 {
+  margin-top: 80px;
+}
+
+.second-container {
+  background-image: url('./images/pattern_bg.png');
+  background-repeat: repeat;
+  background-color: rgb(39, 42, 50);
+  color: white;
+}
+
+.second-container .about-heading::after {
+  content: "";
+  border: 1px solid #ec5242;
+  display: block;
+  height: 2px;
+  width: 50%;
+  margin: 0 auto;
+}
+
+.speaker-section .speaker-title::after {
+  content: "";
+  border: 1px solid #ec5242;
+  display: block;
+  height: 1px;
+  width: 20%;
+  margin: 5px auto;
+}
+
+footer h3::after {
+  content: "";
+  border: 1px solid #ec5242;
+  display: block;
+  height: 1px;
+  width: 3%;
+  margin: 5px auto;
+}
+
+.second-container .program h3 {
+  color: #ec5242;
+}
+
+.second-container .apply-btn {
+  background: #ec5242;
+  text-decoration: none;
+  font-family: 'Cocogoose', sans-serif;
+  color: white;
+  padding: 12px;
+}
+
+.second-container .btn {
+  margin-bottom: 10px;
+}
+
+.second-container .programs .program .cover-gradient-program .img-fluid img {
+  width: 100%;
+  height: auto;
+}
+
+.cover-gradient-program {
+  background: linear-gradient(to top, rgb(102, 100, 100), #535050);
+  opacity: 0.7;
+  margin: 10px 0;
+  width: auto;
+}
+
+#close-ham-img {
+  width: 25px;
+  margin: 45px 0;
+}
+
+.speaker-section .speakers .speaker-info {
```

Comment:
- [ ] This speaker section looks very good. We only need to add margins/paddings to the left and to the right or reduce the width of the `div` with the class of `.speakers` and give a margin value of `margin: 0 auto` to center it within the page.

The here is that the implementation stretches the whole width of the page. Try and work on this.

| The Provide Design |
| --- |
| ![featured-design](https://user-images.githubusercontent.com/45002654/133673712-9602afd7-3ae2-4700-be39-5b04ce6c3a9c.PNG) |

| Your implementation |
| --- |
| ![featured-imp](https://user-images.githubusercontent.com/45002654/133673591-f56c172f-c0a9-428e-a8fa-91a767e1833e.PNG) |